browser/brave_ads/ads_service_impl.cc

```diff
@@ -51,7 +51,6 @@
 #include "brave/components/brave_ads/browser/ads_p2a.h"
 #include "brave/components/brave_ads/browser/ads_storage_cleanup.h"
 #include "brave/components/brave_ads/browser/frequency_capping_helper.h"
-#include "brave/components/brave_ads/browser/notification_helper.h"
 #include "brave/components/brave_ads/browser/service_sandbox_type.h"
```

Comment:
how about moving this into `browser` and dropping `content/public/browser` dep from `component`?